src/DataTable/index.jsx

```diff
@@ -1,76 +1,93 @@
-import React from 'react'
-
-import Pagination from './Pagination'
-import Row from './Row'
-import Search from './Search'
+import React, { Fragment } from "react";
+import { getTotalNumberOfPages, search } from "./logic";
+import PaginationBox from "./PaginationBox";
+import Table from "./Table";
+import SearchBox from "./SearchBox";
 
 class DataTable extends React.Component {
   state = {
-    rows: this.props.rows,
```

Comment:
never pass props to state directly, don't [ Initializing State from Props](https://daveceddia.com/where-initialize-state-react/#:~:text=Don't%20%E2%80%9Ccopy%20props%20into,keep%20it%20up%20to%20date.)